ArduPlane/mode_qstabilize.cpp

```diff
@@ -42,5 +42,8 @@ void ModeQStabilize::update()
 
     plane.nav_roll_cd = (plane.channel_roll->get_control_in() / 4500.0) * roll_limit;
     plane.nav_roll_cd = constrain_int32(plane.nav_roll_cd, -roll_limit, roll_limit);
+
+    // set rudder using stick
+    plane.steering_control.rudder = plane.channel_rudder->get_control_in();
```

Comment:
This would overdrive/destablize the quad tuning on a lot of quadplanes, and has a huge impact based on what the wind over the surface is, I don't think this is a safe change.

Reply:
Particular example being that on my system the rudder is normally only used for ~15% of it's output range, but if you mapped the input directly onto that it would have much larger yaw moments then the quad tuning is expecting, but it would be highly dependent upon the wind (really forward airspeed)

Reply:
@WickedShell Thanks for commenting!
Definitely need to address stability when zooming along at 15 m/sec in Qmodes...
I have another PR which implements gain scaling for tailsitters to deal with high airspeed: #11195 
but I haven't yet thought about whether that can work for non-tailsitters.
I found I was unable to turn without rudder though

Reply:
As I think I described to you we do the wrong thing with the rudder in some modes and nothing in others, so I do want to see us use it. It's just that this won't work out well for existing setups and actively hurts quadplanes that are currently tuned. If it's restricted to QAcro it wouldn't bother me as much, but this would have a very negative impact in flying current quadplane tuning through the full range of winds the aircraft can otherwise handle.

Reply:
I changed the rudder control to motors->get_yaw
That is the demand from the copter controller, and it tests OK in SITL